tests/core/data_layer/test_data_rpc.py

```diff
@@ -39,25 +32,19 @@ async def test_create_insert_get(chia_root: ChiaRoot, wallet_node):
     wallet_node, server_2 = wallets[0]
     wallet = wallet_node.wallet_state_manager.main_wallet
     ph = await wallet.get_new_puzzlehash()
-
     await server_2.start_client(PeerInfo(self_hostname, uint16(server_1._port)), None)
-
     for i in range(0, num_blocks):
         await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
     print(f"confirmed balance is {await wallet.get_confirmed_balance()}")
     print(f"unconfirmed balance is {await wallet.get_unconfirmed_balance()}")
-
-    data_layer = DataLayer(root_path=root, wallet_node=wallet_node)
-    connection = await aiosqlite.connect(data_layer.db_path)
-    data_layer.connection = connection
-    data_layer.db_wrapper = DBWrapper(data_layer.connection)
-    data_layer.data_store = await DataStore.create(data_layer.db_wrapper)
-    data_layer.initialized = True
-    rpc_api = DataLayerRpcApi(data_layer)
+    rpc_api = WalletRpcApi(wallet_node)
+    res = await rpc_api.start_data_layer()
+    assert res["result"] is True
     key = b"a"
     value = b"\x00\x01"
     changelist: List[Dict[str, str]] = [{"action": "insert", "key": key.hex(), "value": value.hex()}]
     res = await rpc_api.create_kv_store()
```

Comment:
@matt-o-how fails for not having enough balance when creating the store
although i farm a few blocks in the start and confirm the main wallet balance 

im probably not funding the correct wallet but it looks like you are using the main one in the data_layer_walet